exporter/src/main.rs

```diff
@@ -52,7 +52,8 @@ struct Opt {
     #[clap(name = "output")]
     output_path: Option<PathBuf>,
 
-    /// Number of frames to capture per file
+    /// Number of frames to capture per file.
+    /// If set to 0, all frames in the SWF's main timeline will be captured
```

Comment:
It makes more sense for me to pass `--frames all` instead of `--frames 0` to capture all frames. It looks non-intuitive

Reply:
I've done some research and hope I've found the right way to implement this.
I need to run a few more tests, but I've raised this as #20930.